sphinx_wagtail_theme/layout.html

```diff
@@ -34,14 +35,22 @@
     {%- if favicon %}
         <link rel="shortcut icon" href="{{ pathto('_static/' + favicon, 1) }}" />
     {%- endif %}
+
+    {#- Include our own stylesheets -#}
+    {%- if sphinx_version_info[0] < 4 %}
     <link rel="stylesheet" type="text/css" href="{{ pathto('_static/' + style, 1) }}" />
+    {%- endif %}
     <link rel="stylesheet" type="text/css" href="{{ pathto('_static/dist/fontawesome.css', 1) }}" />
+    {#- Include stylesheets from sphinx (e.g. user custom css #}
     {%- for css in css_files %}
+    {#- Block pygments css; we are manually including ours in theme.css instead #}
+    {%- if css not in ["_static/pygments.css"] %}
```

Comment:
Correct. Similar to comment above, our theme is NOT using the pygment styles. It is using its own "hard-coded" sass/css style for syntax highlighting. If we include pygments like a normal theme, it will override and mess up the custom style.

Reply:
Thanks for clarifying!